pytrip/ctx.py

```diff
@@ -102,7 +102,7 @@ def create_dicom(self):
 
         ds.KVP = ''  # KVP tag 0x0018, 0x0060
 
-        ds.AcquisitionNumber = ''  # AcquisitionNumber tag 0x0020, 0x0012
+        ds.AcquisitionNumber = 1  # AcquisitionNumber tag 0x0020, 0x0012
```

Comment:
I've checked type of `ds.AcquisitionNumber` here https://www.dabsoft.ch/dicom/6/6/#(0020,0012)

```(0020,0012)	Acquisition Number	AcquisitionNumber	IS	1```

It is IS which means IntegerString (following ftp://dicom.nema.org/medical/DICOM/2013/output/chtml/part05/sect_6.2.html):

```
A string of characters representing an Integer in base-10 (decimal), shall contain only the characters 0 - 9, with an optional leading "+" or "-". It may be padded with leading and/or trailing spaces. Embedded spaces are not allowed.
The integer, n, represented shall be in the range:
-231<= n <= (231-1).
```

I think is it much safer to use: `ds.AcquisitionNumber = '1'` instead of `ds.AcquisitionNumber = 1`